readme-mo.md

```diff
@@ -0,0 +1,24 @@
+## HTTP and the like
+![http1 vs http2](https://i.imgur.com/qPh7wFb.gif)
+
+## General
+
+This is a project that is trying to use HTTP/HTTP2 on the back-end.
+
+The project has been initialised using Yarn.
```

Comment:
For this project it's pretty important to know it uses yarn. To make sure people know that to use you could highlight this using `Yarn` or [yarn].

[yarn]: https://yarnpkg.com/lang/en/